pkg/apiserver/registry/core/shoot/strategy.go

```diff
@@ -143,6 +143,11 @@ func mustIncreaseGeneration(oldShoot, newShoot *core.Shoot) bool {
 				} else {
 					mustIncrease, mustRemoveOperationAnnotation = true, false
 				}
+
+			case v1beta1constants.ShootOperationForceInPlaceUpdate:
+				// The annotation will be removed later by gardenlet once the in-place update is finished.
+				// The generation will be increased if there really is a spec change in the object.
+				mustIncrease, mustRemoveOperationAnnotation = false, false
```

Comment:
Why do we need this? 👀

Reply:
I included this case just to explicitly document the behavior for this value. The default behavior already covers this case. I can also remove it if you want.